selfdrive/athena/athenad.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python3
+from __future__ import annotations
```

Comment:
is this necessary?

Reply:
Yeah, the type hints for Queue don't work correctly without it